src/stress/memory/churn.spec.ts

```diff
@@ -0,0 +1,17 @@
+export const description = `
+Stress tests covering robustness in the presence of heavy buffer and texture
+memory churn.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('churn')
+  .desc(
+    `Allocates and populates a huge number of buffers and textures over time,
+retaining some while dropping or explicitly destroying others. When finished,
+verifies the expected contents of any remaining buffers and textures.`
```

Comment:
Actually, reading this description, I think @sunnyps's test probably is closer to this one than to the buffer_allocation one, since it checks contents at the end. Except this one does both buffers and textures.

Reply:
ack